demo/src/app/components/accordion/demos/basic/basic.component.html

```diff
@@ -1,17 +1,11 @@
-<p>
-  <button type="button" class="btn btn-primary btn-sm" (click)="isOpen = !isOpen">Toggle last panel</button>
-  <button type="button" class="btn btn-primary btn-sm" (click)="firstDisabled = !firstDisabled">Enable / Disable first panel</button>
-  <button type="button" class="btn btn-danger btn-sm" (click)="removeDynamic()">Remove last dynamic</button>
-</p>
-
-<ngb-accordion close-others="true">
-  <ngb-panel title="This is the header" [disabled]="firstDisabled" open="true">
-    This is the content
+<ngb-accordion>
+  <ngb-panel title="Collapsible Group Item #1" open="true">
+    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably haven't heard of them accusamus labore sustainable VHS.
```

Comment:
just a nit, would you mind wrapping these at like 120 or so?  i'd like the demo HTML to fit without scrolling if entirely possible.